### ISSUE TITLE
sass compiling with MAMP using terminal

### ISSUE THREAD

**Issue by MichelleDiamond** [getgrav/grav#1369]
Hello,

I cannot get a custom.sass file to compile, or a normal custom.css file (already available with helium) to work.

I am not a developer. I have gone through the docs. I did try to create my own theme but then the admin panel suddenly had 0 plugins so I deleted the new theme folder.

Please help me in as simple terms as possible.

Thanks

**Comment by rhukster** [comment]
What theme are you using? What steps did you perform to compile the sass? What process/steps/tutorial did you follow to create your own theme?

I'de be glad to help, but you need to give me as much information as possible. Also, you might find it more helpful to ask these kinds of questions in the Slack chat where things are more 'interactive' . The issues list is mainly for bugs and other incompatibilities.

Cheers.

**Comment by MichelleDiamond** [comment]
Hi Andy,

I have been through so many tutorials that I cannot remember at all as to how I got to creating a theme. 

I have just deleted the first install and have now unpacked a new helium package into MAMP. Im busy setting it up. Previous was a mess..

I think I have managed to install SASS. Following through from the site referred to in the grav docs. I was, however, not sure if I needed to install into the site folder or if I was to just add the command as per instruction. See the terminal output below.


Basically, I have no idea how to start with SASS using Grav. I don’t want to write css because, from experience, it can be disordered and cluttered over time.

I have my system set up to run ionic, angular using gulp and SourceTree. The app source files are configured and I simply follow the read.md file. The developer who uses me walks me through the set up process and then I fly.

———

Dreamweaver has an auto compiler so I tried setting up the local site. I considered instilling compass via Dreamweaver but then was not sure of where the grave paths were.

Would really appreciate a bit of help with the css setup. I will join the slack group. Apologies for any inconvenience.

Example below

.http://sass-lang.com/guide <http://sass-lang.com/guide>  eg:  I have tried various options to get the below to work.

sass --watch app/sass:public/stylesheets
I don’t know how to find the path to /Applications/MAMP/htdocs/grav-skeleton-helium-site

———————

Last login: Tue Mar 21 10:21:44 on console
You have mail.
Michelles-iMac:~ michellediamond$ sass -v
-bash: sass: command not found
Michelles-iMac:~ michellediamond$ gem install sass
Fetching: sass-3.4.23.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.0.0 directory.
Michelles-iMac:~ michellediamond$ sudo gem install sass
Password:
Fetching: sass-3.4.23.gem (100%)
ERROR:  While executing gem ... (Errno::EPERM)
    Operation not permitted - /usr/bin/sass
Michelles-iMac:~ michellediamond$ sudo gem install -n /usr/local/bin sass
Successfully installed sass-3.4.23
Parsing documentation for sass-3.4.23
Installing ri documentation for sass-3.4.23
1 gem installed
Michelles-iMac:~ michellediamond$ sass --watch app/sass:Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss/custom.scss 
>>> Sass is watching for changes. Press Ctrl-C to stop.
  directory Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss
      error No such file or directory - app/sass
Ignoring ffi-1.9.6 because its extensions are not built.  Try: gem pristine ffi-1.9.6
^CMichelles-iMac:~ michellediamond$ sass --watch/Applications/MAMP/htdocs/grav-sleton-helium-site/user/themes/g5_helium/scss/custom.scss 
OptionParser::InvalidOption: invalid option: --watch/Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss/custom.scss
  Use --trace for backtrace.
Michelles-iMac:~ michellediamond$ sass --watch Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss/custom.scss 
>>> Sass is watching for changes. Press Ctrl-C to stop.
      write Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss/custom.css
      write Applications/MAMP/htdocs/grav-skeleton-helium-site/user/themes/g5_helium/scss/custom.css.map
Ignoring ffi-1.9.6 because its extensions are not built.  Try: gem pristine ffi-1.9.6
^CMichelles-iMac:~ michellediamond$ sass watch
Errno::ENOENT: No such file or directory - watch
  Use --trace for backtrace.
Michelles-iMac:~ michellediamond$ grav gpm
-bash: grav: command not found
Michelles-iMac:~ michellediamond$ grav gpm install
-bash: grav: command not found
Michelles-iMac:~ michellediamond$ bin/grav list
-bash: bin/grav: No such file or directory
Michelles-iMac:~ michellediamond$ bin/grav help install
-bash: bin/grav: No such file or directory
Michelles-iMac:~ michellediamond$ /Applications/MAMP/htdocs/grav-skeleton-helium-site/bin/grav list
FATAL: Must be run from ROOT directory of Grav!Michelles-iMac:~ michellediamond$
